src/screens/Crypto/Crypto.tsx

```diff
@@ -53,15 +52,17 @@ const Crypto = ({ navigation }: Props) => {
   return (
     <SafeAreaView>
       <Background>
-        <ArrowButton onPress={() => navigation.goBack()} >
-          <IconButton source={require('../../assets/icons/left.png')} />
+        <ArrowButton onPress={navigation.goBack.bind(null)} >
```

Comment:
I think if you put only navigation.goBack only. It will work finr. In this case is onlh a callback with no props